runnumber-rest/api.py

```diff
@@ -0,0 +1,110 @@
+import os
+from datetime import datetime
+import flask
+from flask_restful import Api, Resource
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import func
+
+__all__ = ["app", "api", "db"]
+
+app = flask.Flask(__name__)
+
+app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
+    "DATABASE_URI", "sqlite:////tmp/test.sqlite"
+)
+
+db = SQLAlchemy(app)
+api = Api(app)
+
+from database import RunNumber
+from authentication import auth
+
+@api.resource("/runnumber/get")
```

Comment:
The run registry has snippets that can be used to test, can you add those here?

Reply:
Added in next commit